### DISCUSSION TITLE
Remote start/stop

### DISCUSSION THREAD

**Discussion by drc38** [lbbrhzn/ocpp#111] [Q&A] [answered]
@lbbrhzn, @jeremycoppey when you stop your chargers using the Charge_control switch what state does the charger go to? Can you then remotely start charging again? If so what state transitions happen before charging starts?

**Comment by lbbrhzn** [comment]
I haven't tried to stop and start during a charging session yet. I'll give it it try during my next charging session.

**Reply by drc38** [reply]
@Strixx76 can you report back on what your charger does in response to the above, cheers?

**Comment by Strixx76** [comment, marked as answer]
This is the full flow of states I have noticed for the sensor `charger_status_connector`. (Just tried a full cycle again)

**Start**
`Available`
Cable plug in
`Preparing`
switch.charger_charge_control -> On
`SuspendedEV`
about 5-10 sec later
`Charging`
_During the session when battery is almost full I can get `SuspendedEV` and `SuspendedEVSE`._

**Stop**
switch.charger_charge_control -> Off (When in the middle of a session)
`Finishing`
_I have only once let the car stay connected after stopped the session and then it state turns to `Preparing` by it self after about 8 hours._

**Start again**
switch.charger_charge_control -> On
`Preparing`
`SuspendedEV`
`Charging`

On page 41 in [Open Charge Point Protocol 1.6](https://github.com/mobilityhouse/ocpp/blob/master/docs/v16/ocpp-1.6.pdf) you can se all possible state transitions according to the standard.


BTW. I have noticed that not all states shows in the Logbook, but I can see them in the History. Is this is HA or component bug?

**Reply by drc38** [reply]
Thanks,  I'm aware of the state transitions table. My charger will stop and go to `Finishing` however I cannot start it again by setting switch.charger_charge_control -> On (ie send a `RemoteStartTransaction`)

I take it you have no issues with that transition?

Not sure how HA determines what gets logged from the integration.

**Reply by Strixx76** [reply]
No problems starting again after stopped.
